app/schemas/tech.relaycorp.courier.data.database.AppDatabase/1.json

```diff
@@ -2,24 +2,42 @@
   "formatVersion": 1,
   "database": {
     "version": 1,
-    "identityHash": "4056a2f52fe6f834146b1ab85d35e7a8",
+    "identityHash": "011b4f5a90e3c27ac4a0d9daf6061ec3",
     "entities": [
       {
-        "tableName": "cargo",
-        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`recipientAddress` TEXT NOT NULL, `messageId` TEXT NOT NULL, `creationTime` INTEGER NOT NULL, `expirationTime` INTEGER NOT NULL, `storagePath` TEXT NOT NULL, `size` INTEGER NOT NULL, `senderAddress` TEXT NOT NULL, PRIMARY KEY(`recipientAddress`, `messageId`))",
+        "tableName": "Message",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`recipientAddress` TEXT NOT NULL, `recipientType` TEXT NOT NULL, `senderAddress` TEXT NOT NULL, `messageId` TEXT NOT NULL, `messageType` TEXT NOT NULL, `creationTime` INTEGER NOT NULL, `expirationTime` INTEGER NOT NULL, `storagePath` TEXT NOT NULL, `size` INTEGER NOT NULL, PRIMARY KEY(`senderAddress`, `messageId`))",
         "fields": [
           {
             "fieldPath": "recipientAddress",
             "columnName": "recipientAddress",
             "affinity": "TEXT",
             "notNull": true
           },
+          {
+            "fieldPath": "recipientType",
+            "columnName": "recipientType",
+            "affinity": "TEXT",
+            "notNull": true
+          },
+          {
+            "fieldPath": "senderAddress",
+            "columnName": "senderAddress",
+            "affinity": "TEXT",
+            "notNull": true
+          },
           {
             "fieldPath": "messageId",
             "columnName": "messageId",
             "affinity": "TEXT",
             "notNull": true
           },
+          {
+            "fieldPath": "messageType",
+            "columnName": "messageType",
+            "affinity": "TEXT",
+            "notNull": true
+          },
```

Comment:
Similarly, if we can mimic an enum here that'd be great -- e.g., `CHECK( messageType IN ('cargo','cca') )`

Reply:
See previous reply on the same issue.